ee/api/sentry_stats.py

```diff
@@ -107,9 +107,11 @@ def sentry_stats(request: HttpRequest):
         target_end_date = current_time.strftime("%Y-%m-%dT%H:%M:%S")
         target_start_date = (current_time - timedelta(days=1)).strftime("%Y-%m-%dT%H:%M:%S")
 
-        data, total_count = get_sentry_stats(target_start_date, target_end_date)
+        _, total_count = get_sentry_stats(target_start_date, target_end_date)
```

Comment:
this function raises a ValidationError when the integration is disabled. There's 2 settings to it, `SENTRY_AUTH_TOKEN` and `SENTRY_ORGANIZATION`. 

Mind try-catching here instead to get the result of `sentry_integration_enabled`, or moving all conditions out of the above?